src/jobs/deploy.yml

```diff
@@ -58,6 +65,11 @@ parameters:
     type: boolean
     description: Specify if deploy command hould return a zero exit code if there are no changes to be made to the stack.
     default: true
+  machine-image:
+    default: ubuntu-2004:202010-01
```

Comment:
Instead of repeating the same image over and over, can we just point to an executor?

Reply:
Open to discussing this, I removed the executor as it only wrapped the machine executor. This actually is somewhat relevant to our conversation last week. This would be more acceptable if the user could use Machine: true, but maybe since we are phasing out the practice, this _should_ have an executor?

Reply:
While I agree that executors should ideally be reserved for orbs with unique images, I think that the ability to conserve a "default, known good" executor or image is a good idea/shortcut for orbs.

Reply:
👍 
Adding it back.

We may be able to setup renovateapp to automatically PR when the machine image updates.